lib/archive/archiver.go

```diff
@@ -47,6 +51,7 @@ type FilterSpec struct {
 	Exclusions map[string]struct{}
 	RebasePath string
 	StripPath  string
+	TargetPath string
```

Comment:
Did we determine that this was a value-add over just using `Rebasepath`? I still think they're the same behavior.